src/network/networkd-network.h

```diff
@@ -310,6 +310,7 @@ struct Network {
         Hashmap *prefixes_by_section;
         Hashmap *route_prefixes_by_section;
         Hashmap *rules_by_section;
+        OrderedHashmap *dhcp_static_leases_by_section;
```

Comment:
Why do you use `OrderedHashmap`?? I do not think order of the sections is not important.

Reply:
No, that's user's mis-configuration. `OrderedHashmap` should not be used to handle mis-configurations.